sense/loading.py

```diff
@@ -0,0 +1,174 @@
+                             if config.model_name == requested_model_name]
+    if requested_version:
+        model_config_list = [config for config in model_config_list
+                             if config.version == requested_version]
+
+    # Check if not empty
+    if not model_config_list:
+        raise Exception(f'Could not find a model configuration matching requested parameters:\n'
+                        f'\tmodel_name={requested_model_name}\n'
+                        f'\tversion={requested_version}')
+
+    for model_config in model_config_list:
+        path_weights = model_config.get_path_weights()
+        path_weights_string = json.dumps(path_weights, indent=4, sort_keys=True)  # used in prints
+
+        if all(os.path.exists(path) for path in path_weights.values()) or using_travis():
+            print(f'Weights found:\n{path_weights_string}')
+            weights = {}
+            for name, path in path_weights.items():
+                load_fn = load_backbone_weights if name == 'backbone' else load_weights
+                weights[name] = load_fn(path)
+
+            return model_config, weights
+        else:
+            print(f'Could not find at least one of the following files:\n{path_weights_string}')
+
+    raise Exception('ERROR - Weights file missing. To download, please go to '
+                    'https://20bn.com/licensing/sdk/evaluation and follow the '
+                    'instructions.')
+
+
+def load_weights(checkpoint_path: str):
+    """
+    Load weights from a checkpoint file.
+
+    :param checkpoint_path:
+        A string representing the absolute/relative path to the checkpoint file.
+    """
+    return torch.load(checkpoint_path, map_location='cpu')
+
+
+def load_weights_from_resources(checkpoint_path: str):
+    """
+    Load weights from a checkpoint file located in the resources folder.
+
+    :param checkpoint_path:
+        A string representing the absolute/relative path to the checkpoint file.
+    """
+    checkpoint_path = os.path.join(RESOURCES_DIR, checkpoint_path.split(f'resources{os.sep}')[-1])
+    try:
+        return load_weights(checkpoint_path)
+
+    except FileNotFoundError:
+        raise FileNotFoundError('Weights file missing: {}. '
+                                'To download, please go to '
+                                'https://20bn.com/licensing/sdk/evaluation and follow the '
+                                'instructions.'.format(checkpoint_path))
+
+
+def load_backbone_weights(checkpoint_path: str):
+    """
+    Load backbone weights from a checkpoint file, unless Travis is used. Raises an error pointing
+    to the SDK page in case weights are missing.
+
+    :param checkpoint_path:
+        A string representing the absolute/relative path to the checkpoint file.
+    """
+    if not using_travis():
+        return load_weights_from_resources(checkpoint_path)
+    else:
+        print('Weights are not loaded on Travis.')
```

Comment:
Is it really necessary to have this method wrap `load_weights_from_resources`?

Reply:
It is useful at the time where I write this. Integration tests would complain on Travis without this. But maybe this can go when we refactor `get_relevant_weights`